recipes/pyjion/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "pyjion" %}
+{% set version = "1.2.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tonybaloney/Pyjion.git
+  git_tag: {{ version }}
+
+
+build:
+  skip: True  # [py < 310]
+  number: 0
+  script: |
+    git submodule update --init --recursive
+    {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - pyjion = pyjion.__main__:main
+
+requirements:
+  build:
+    - cmake
```

Comment:
Do you not need a compiler?

Reply:
It has `dotnet` 🤦

Reply:
It does, but I'm not sure if it also requires a C or C++ compiler. From the languages, I think it probably does:

![image](https://user-images.githubusercontent.com/881019/152682226-b8b6ddd0-12f5-4668-921c-6347009be25e.png)

https://github.com/tonybaloney/Pyjion

Reply:
![image](https://user-images.githubusercontent.com/881019/152682495-d05690b3-b738-4707-8e20-71b90f40fc9e.png)
https://pyjion.readthedocs.io/en/latest/development.html#compiling-from-source

Suggests you need VS2019 (on Windows I guess). Fortunately, it seems VS2019 support was added in [`scikit-build` 0.12.0](https://github.com/scikit-build/scikit-build/releases/tag/0.12.0) whcih is [available on CF](https://github.com/conda-forge/scikit-build-feedstock).

Reply:
I'll unresolve so @tonybaloney doesn't miss this question. Otherwise, I imagine CI will tell us pretty quickly if a compiler is missing!